airgun/entities/oscaptailoringfile.py

```diff
@@ -100,3 +100,21 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+
+
+@navigator.register(OSCAPTailoringFileEntity, 'Delete')
+class DeleteSCAPTailoringFile(NavigateStep):
+    """To get data from SCAPPolicyDetail view
```

Comment:
is the delete view is different from edit view? if not just add a method instead of writing a new view e.g. https://github.com/SatelliteQE/airgun/blob/389b9d1286f0de4f9ca40f02b1c9d4da2d457fc1/airgun/entities/hostgroup.py#L38

Reply:
Thanks, will update PR.